packages/manager/modules/product-offers/src/product-offers.html

```diff
@@ -0,0 +1,108 @@
+<oui-stepper
+    data-current-index="$ctrl.workflow.currentIndex"
+    data-on-finish="$ctrl.workflow.validateOffer()"
+>
+    <!-- Optional Configuration Step -->
+    <div data-ng-transclude></div>
+
+    <!-- Pricing -->
+    <oui-step-form
+        data-header="{{:: 'ovh_product_offers_pricing_header' | translate }}"
+        data-editable="$ctrl.workflow.pricings.length > 1 && !$ctrl.workflow.isLoading()"
+        data-valid="$ctrl.workflow.pricing"
+        data-prevent-next="true"
+        data-on-focus="$ctrl.workflow.getPricings()"
+        data-loading="$ctrl.workflow.loading.getPricingInformation"
+    >
+        <div class="mb-3" data-ng-if="$ctrl.workflow.hasUniquePricing()">
+            <span
+                data-ng-pluralize
+                data-count="$ctrl.workflow.pricings[0].interval"
```

Comment:
Is this pricing is fetch by `capacities` in the controller ? If not it definitely should

Reply:
https://github.com/ovh/manager/pull/2214/files#diff-72de6a8dce68e993fc9e87efc5250354R100 ;)